docs/resources/guides/deployment/bare_metal.rst

```diff
@@ -86,7 +93,7 @@ default. You can start the server by enabling the unit.
    $ sudo systemctl enable --now gel-server-6
 
 This will start the server on port 5656, and the data directory will be
-``/var/lib/gel/1/data``.
+``/var/lib/gel/6/data``.
```

Comment:
For versions 6.0 and 6.1 this is a lie. The directory is `/var/lib/edgedb/6/data` for those versions, but I'm not sure it is worth adding an explanation here. The directory will be correct starting in version 6.2.

Reply:
Seems fine to pretend we always intended this and the old behavior was an unintentional defect.

Reply:
follow up issue https://github.com/geldata/gel/issues/8440